# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../tea/hotfix.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- tea-16.0.4/src/callbacks.c.vanilla	2007-03-02 09:34:38.000000000 +0000
+++ tea-16.0.4/src/callbacks.c	2007-03-02 09:35:06.000000000 +0000
@@ -25,7 +25,7 @@
 
 #ifdef PRINTING_SUPPORTED
 
-#include <gtk-unix-print-2.0/gtk/gtkprintunixdialog.h>
+#include <gtk/gtkprintunixdialog.h>
 
 #endif
 
